pkg/daemon/daemon.go

```diff
@@ -207,6 +207,14 @@ func (dn *Daemon) Run(stopCh <-chan struct{}, exitCh <-chan error) error {
 	for {
 		select {
 		case <-stopCh:
+			// clean files from host if we are running in systemd mode
+			if vars.UsingSystemdMode {
+				err := systemd.CleanSriovFilesFromHost(vars.ClusterType == consts.ClusterTypeOpenshift)
```

Comment:
so when the daemon is stopped we delete systemd files ?

if the node reboots (given pods get stop signal) wont it delete the files ? which will in turn mean the node will not have SR-IOV configured on startup  ?

Reply:
What you say is correct. We should have a way to identify whether a `reboot` has been requested. In that case the cleanUp has to be a best effort because we can have an ongoing reboot and also a `spec` change in the `daemonSet`.